4/john-wiegley-captions.srt

```diff
@@ -2309,17 +2309,18 @@ That's not really what I want.
 
 534
 00:45:07,300 --> 00:45:09,800
-I want to see the
-result of every line as I type.
+I want to have my own forever because I
+want to see the result of every line as I
+type.

```

Comment:
"I want to have my own forever" sounds like something the villain in a fairy story would say.